src/app/qgsmaptoolsimplify.h

```diff
@@ -71,12 +72,15 @@ class APP_EXPORT QgsMapToolSimplify: public QgsMapToolEdit
     QString statusText() const;
 
   public slots:
-    /** Slot to change display when slidebar is moved */
+
+    /**
+     * Slot to change display when slidebar is moved */
     void setTolerance( double tolerance );
 
     void setToleranceUnits( int units );
 
-    /** Slot to store feture after simplification */
+    /**
+     * Slot to store feture after simplification */
```

Comment:
Fixed in https://github.com/qgis/QGIS/commit/02ea2d77036e929c478600172e564727598f11f3